src/toolchain/export-web.md

```diff
@@ -0,0 +1,155 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more difficult to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+However, setting up a web server to host and share your game is considered out of scope of this guide, and is best explained elsewhere.
+
+Information to consider before starting:
```

Comment:
This should no longer be necessary, the yellow warning box draws enough attention.

Reply:
Line 14?

Reply:
Yes, only this part:
```
Information to consider before starting:
```